docs/reference/organizations-on-nuget-org.md

```diff
@@ -0,0 +1,105 @@
+
+## Managing organization members
+
+As the organization administrator, you can add members by providing each member's nuget.org *user account name*; email addresses cannot be used. You then mark each member as a collaborator or administrator with the following permissions:
+
+| Permission | Collaborator | Administrator |
+| --- | --- | --- |
+| Manage the organization's packages<br/>(submit new packages, update or unlist existing packages) | Yes | Yes |
+| Change organization metadata<br/>(email address, notification settings) | No | Yes |
+| Manage organization members | No | Yes |
+| Request or act on co-ownership requests for organization packages | No | Yes |
+
+## Managing packages
+
+As an organization member, you can see all the organization's packages in the **Packages** section of the organization's details page. On the **Manage Packages** page you can also view packages across your account and all organizations of which you're a member. You can select the account filter to view packages by your account or any specific organization.
+
+![Managing packages with the account filter](media/org-manage-packages-option.png)
+
+## Publishing packages
+
+You publish packages to an organization like you publish packages to a user account, by directly uploading the package to nuget.org or by pushing the package through the `nuget push` or `dotnet nuget push` CLI commands.
+
+### Uploading packages
+
+When you directly upload a new package on the [nuget.org Upload](https://www.nuget.org/packages/manage/upload) page, you assign the package owner to a user or organization account :
+
+![Upload package with account option](media/org-upload-option.png)
+
+When submitting updates to an existing package, you don't see the account dropdown because package ownership is already known to nuget.org.
+
+### Using API keys
+
+To push a package through the `nuget push` or `dotnet nuget push` CLI commands, you must obtain the API key needed by those commands. For details, see [Publish a package](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio#publish-the-package).
+
+When creating a new API key, select the appropriate organization in the **Package Owner** drop down. Any API key you create is applicable only to the chosen organization:
+
+![API key with account option](media/org-apikey-option.png)
+
+## Removing an organization
+
+As a user, you can remove yourself from an organization by selecting the `X` button shown by your organization membership:
+
+![Removing a user account from an organization](media/org-remove-self-option.png)
+
+Administrators can remove any member from the organization, including other administrators. If you're the sole administrator for an organization, you cannot remove yourself unless you first add another member as an administrator.
+
+You can delete an organization if the following criteria are met:
```

Comment:
This is not going live with the feature. It will probably come in the deployment afterwards. You should remove this and add it back when we add the feature.